src/python_testing/TC_JFDS_2_3.py

```
@@ -33,6 +33,7 @@
 #     quiet: true
 # === END CI TEST ARGUMENTS ===
 
+import asyncio
```

Flake8 (7.3.0)

[error] 'asyncio' imported but unused

(F401)